diff --git a/neutron/common/config.py b/neutron/common/config.py
index b6ba1a7..e99bb07 100644
--- a/neutron/common/config.py
+++ b/neutron/common/config.py
@@ -192,7 +192,6 @@ logging.register_options(cfg.CONF)
 
 def init(args, **kwargs):
     cfg.CONF(args=args, project='neutron',
-             version='%%(prog)s %s' % version.version_info.release_string(),
              **kwargs)
 
     # FIXME(ihrachys): if import is put in global, circular import
@@ -213,9 +212,6 @@ def setup_logging():
     product_name = "neutron"
     logging.setup(cfg.CONF, product_name)
     LOG.info(_LI("Logging enabled!"))
-    LOG.info(_LI("%(prog)s version %(version)s"),
-             {'prog': sys.argv[0],
-              'version': version.version_info.release_string()})
     LOG.debug("command line: %s", " ".join(sys.argv))
 
 
