--- a/neutron/common/config.py
+++ b/neutron/common/config.py
@@ -198,7 +198,6 @@
 
 def init(args, **kwargs):
     cfg.CONF(args=args, project='neutron',
-             version='%%(prog)s %s' % version.version_info.release_string(),
              **kwargs)
 
     # FIXME(ihrachys): if import is put in global, circular import
@@ -219,9 +218,6 @@
     product_name = "neutron"
     logging.setup(cfg.CONF, product_name)
     LOG.info(_LI("Logging enabled!"))
-    LOG.info(_LI("%(prog)s version %(version)s"),
-             {'prog': sys.argv[0],
-              'version': version.version_info.release_string()})
     LOG.debug("command line: %s", " ".join(sys.argv))
 
 
